pom.xml

```diff
@@ -1023,8 +1023,8 @@
             </dependency>
           </dependencies>
           <configuration>
-            <configLocation>beam/checkstyle.xml</configLocation>
```

Comment:
Do you know why this would be needed? This works fine in Jenkins and on my Desktop.

Java is supposed to pick up these files from the `build-tools` module on the classpath.